docs/index.md

```diff
@@ -2,34 +2,173 @@
+
+**[TODO: Add detail about any extra files that need to be included in the C2M2 datapackage. E.g., DCC and contact info.]**
+
 ### Find your DCC's identifier
 
+When using `cfde-submit` to submit a new C2M2 instance, you must enter your DCC's unique identifier, issued by CFDE-CC. You can find this identifier in the CFDE Portal. Visit the [Onboarded DCC](https://app.nih-cfde.org/chaise/recordset/#registry/CFDE:dcc@sort(RID) page in the CFDE Portal and locate your DCC in the table.
+
+The identifier you need for `cfde-submit` is of the form `cfde_registry_dcc:*`, where the `*` is replaced by a short form of your DCC's name. For example, the DCC identifier for The Human Microbiome Project is `cfde_registry_dcc:hmp`.
+
+**[TODO: The link above will not work until the production portal has been updated. When the production portal is updated, make sure it works!]**
+
+### Check your system
+
+Here are a few things that could make `cfde-submit` not work correctly on your system.
+
+If you have [Globus Connect Personal](https://www.globus.org/globus-connect-personal) installed on your computer, `cfde-submit` will attempt to use it to transfer your C2M2 instance to CFDE.
+
+* Make sure Globus Connect Personal is running. If it is installed but not running, `cfde-submit` will start a transfer, but the transfer will not begin until you start Globus Connect Personal.
+* Make sure Globus Connect Personal has access to the folder where your C2M2 instance resides on your system. By default, Globus Connect Personal has access to your home directory on the system and everything within it. If your data is not somewhere within your home directory, or if you have changed the Preferences for Globus Connect Personal to customize its access on your system, there is a good chance `cfde-submit` will not be able to transfer your C2M2 instance. Move the C2M2 instance folder to your home directory or set the Access Preferences in Globus Connect Personal to provide access to the C2M2 instance folder.
+
+---
+
 ## Use the tool
 
+Once you've created a C2M2 instance, the next steps are to submit it to the CFDE Portal, review it, and--assuming it is satisfactory--approve it (or have someone in your DCC approve it) for public viewing. This section walks through the submission process.
+
 ### Login
 
+Submitting a new C2M2 instance for your DCC requires authorization, so the first step when using `cfde-submit` is to login.
+
+> NOTE: Logging in requires a web browser, but it can be done in a remote terminal shell as long as you have a web browser on your local system.
+
+> NOTE: If you use another `cfde-submit` command without being logged in, it will automatically start the login process. We recommend using the login command anyway unless you believe you are logged in already.
```

Comment:
Should we change this behavior? It seems odd that we automatically login users and additionally provide a login command. It seems like we should do one or the other, but not both.